.github/ISSUE_TEMPLATE/4-documentation.yml

```diff
@@ -0,0 +1,23 @@
+name: 📚 Documentation
+description: Report an issue related to https://mmocr.readthedocs.io/en/latest/.
+labels: "docs"
+title: "[Docs] "
+
+body:
+- type: textarea
+  attributes:
+    label: 📚 The doc issue
+    description: >
+      A clear and concise description of what content in https://mmocr.readthedocs.io/en/latest/ is an issue.
```

Comment:
```suggestion
      A clear and concise description of what content in https://mmocr.readthedocs.io/en/latest/ or https://mmocr.readthedocs.io/en/dev-1.x/ is an issue.
```

Reply:
We can also use a single option item to label main/dev-1.x 
An example: https://github.com/open-mmlab/mmediting/pull/1286/files#diff-ffcbbc5f70e05184cd63fe22a18df4180c461ca4cf2dc99bb3251e22a6505aa1R7-R17